scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -20,15 +20,28 @@ class SensorVersionsFailure(Exception):
 
 gkecluster=GKECluster("compat-test")
 
+if len(chart_versions) > 1:
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[0]
+
 failing_sensor_versions = []
 for version in chart_versions:
     os.environ["SENSOR_CHART_VERSION"] = version
     try:
         make_compatibility_test_runner(cluster=gkecluster).run()
     except Exception:
-        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version}")
+        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version} and central version {chart_versions[0]}")
         failing_sensor_versions += version
 
+if len(chart_versions) > 1:
+    os.environ["SENSOR_CHART_VERSION"] = chart_versions[0]
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[1]
+    try:
```

Comment:
`get_latest_release_versions` returns us existing releases. It does **not** return us _current_ version, i.e. the current development build, in its `0`-th element.

Therefore, what we test here is the latest release of sensor against the penultimate release of central. It is not incorrect, but it is already too late to catch any "new sensor"<->"older central" regressions because **both things were already released** (by definition of `get_latest_release_versions`).

I believe, the intent of ROX-16548 is to **prevent** regressions while the code is still in development. That means: we should test the latest release of central against **the current build of sensor**. This way, any regressions introduced in the current code will be causing tests to break thus informing us, engineers, of necessary fixes.

I re-read the Slack thread https://redhat-internal.slack.com/archives/C02HX541YQN/p1681805595490399 and see there's no contradiction.

Please let me know what you think.

Reply:
Yes this makes a lot of sense. Sorry, back when you left your review I just implemented what you asked me to change without replying to each comment and in the process this seems to have gotten lost. The other ones I did though and this one should be done too now, CI is running.